python/examples/13-agent-name-service/agent1.py

```diff
@@ -22,10 +22,12 @@ class Message(Model):
 faucet = get_faucet()
 DOMAIN = "example.agent"
 
+logger.info(f"Adding testnet funds to {my_wallet.address()}...")
 faucet.get_wealth(my_wallet.address())
+logger.info(f"Adding testnet funds to {my_wallet.address()}...complete")
 
 
-@bob.on_event("startup")
+@bob.on_interval(10)
```

Comment:
Need to retry this if the first try is before the agent is registered.

Reply:
Makes sense 👍🏻